scripts/qa_crowbarsetup.sh

```diff
@@ -2303,6 +2315,18 @@ function oncontroller()
     return $?
 }
 
+function get_docker_image()
+{
+    if [ -n "$want_docker" ]; then
```

Comment:
IMHO, this if should not be inside this function. The function should just do what its name suggests, and not depend on another flag.